### PR TITLE
[pipelines] Add support for global values and postgresql 12.x

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

* **Breaking change:** Updated `imagePullSecrets` value from string to list
* Added support for global values
* Updated maintainers in chart.yaml
* Update postgresql tag version to `12.3.0-debian-10-r71`
* Update redis sub chart version to `10.9.0`
* Update postgresql sub chart version to `9.3.4` - [9.x Upgrade Notes](https://github.com/bitnami/charts/tree/master/bitnami/postgresql#900)
* Update rabbitmq sub chart version to `7.4.3` - [7.x Upgrade Notes](https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#to-700)
* Please refer [here](https://github.com/jfrog/charts/blob/master/stable/pipelines/README.md#special-upgrade-notes) for upgrade notes
* **IMPORTANT**
* If this is a new deployment or you already use an external database (`postgresql.enabled=false`), these changes **do not affect you**!
* If this is an upgrade and you are using the default PostgreSQL (`postgresql.enabled=true`), you need to pass previous 9.x's postgresql.image.tag and databaseUpgradeReady=true

### CODE REVIEW

File: stable/pipelines/README.md [hunk 1]
```diff
@@ -81,6 +80,17 @@ kubectl create ns pipelines
 helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f pipelines/values-ingress.yaml -f pipelines/values-ingress-passwords.yaml
 ```
 
+### Special Upgrade Notes
+
+While upgrading from Pipelines 1.x to 2.x and above charts due to breaking rabbitmq (when `rabbitmq.enabled=true`) subchart changes please run,
+
+```bash
+$ kubectl delete statefulsets <old_statefulset_pipelines_name>
```
Comment (danielezer) [inline stable/pipelines/README.md:88]: Use the actual name with only the release name as a placeholder

**Review by danielezer — APPROVED** [review]

**Review by peters95 — CHANGES_REQUESTED** [review]
Please use the customInitBeginContainers in the values.yaml instead of making this change. This change made pipelines chart use both customInitContainer and customInitBeginContainer similar to our other charts.

**Review by peters95 — APPROVED** [review]

**Comment by peters95** [comment]
Updated to have global, pipelines, and vault customInitContainersBegin and customInitContainers.

**Review by peters95 — CHANGES_REQUESTED** [review]
Verified the initContainer and initContainerBegin works successfully but appears that the services are not all coming up.. Please look into this and test these changes more.

johnp@johnp-mac pipelines % kubectl logs pipelines-pipelines-services-0  -c custom-setup

johnp@johnp-mac pipelines % kubectl logs pipelines-pipelines-services-0  -c custom-begin-setup



kubectl get pods
artifactory-ha-artifactory-ha-member-0    1/1     Running            0          23m
artifactory-ha-artifactory-ha-primary-0   1/1     Running            0          23m
artifactory-ha-nginx-544d7c9895-hf9r2     1/1     Running            0          23m
artifactory-ha-postgresql-0               1/1     Running            0          23m
pipelines-pipelines-services-0            2/12    CrashLoopBackOff   9          2m4s
pipelines-pipelines-vault-0               2/2     Running            0          2m4s
pipelines-postgresql-0                    1/1     Running            0          2m4s
pipelines-rabbitmq-0                      1/1     Running            0          2m4s
pipelines-redis-master-0                  1/1     Running            0          2m4s

kubectl describe pod pipelines-pipelines-services-0   
Name:           pipelines-pipelines-services-0
Namespace:      default
Priority:       0
Node:           gke-johnppipelineschartc-default-pool-f2d5c611-lm1b/10.98.44.36
Start Time:     Thu, 08 Oct 2020 10:40:09 -0700
Labels:         app.kubernetes.io/instance=pipelines
                app.kubernetes.io/name=pipelines
                component=pipelines-pipelines-services
                controller-revision-hash=pipelines-pipelines-services-7cd9457599
                statefulset.kubernetes.io/pod-name=pipelines-pipelines-services-0
Annotations:    checksum/configaws: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
                checksum/configfilebeat: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
                checksum/configk8s: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
                checksum/secretaws: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
                checksum/secretdb: fa5a0ac2cace9ab5d8f5b0c17854e5251e37c804a31fcd2d70c1a6cb38342989
                checksum/secretk8s: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
                checksum/systemyaml: e1ce48ccf8d4a6632d525b11c404328b7f81f3dd2ec8c39162bcf86c4e5170a9
                kubernetes.io/limit-ranger:
                  LimitRanger plugin set: cpu request for container router; cpu request for container api; cpu request for container www; cpu request for co...
Status:         Running
IP:             10.4.2.10
Controlled By:  StatefulSet/pipelines-pipelines-services
Init Containers:
  custom-begin-setup:
    Container ID:  docker://4cb64b3f19e85c19ca79ffae0ea06589360667113595d20de6009db6a7c1598f
    Image:         docker.bintray.io/alpine:3.12
    Image ID:      docker-pullable://docker.bintray.io/alpine@sha256:a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65
    Port:          <none>
    Host Port:     <none>
    Command:
      sh
      -c
      touch /opt/jfrog/pipelines/var/etc/example-custom-setup
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:40:10 -0700
      Finished:     Thu, 08 Oct 2020 10:40:10 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:        100m
    Environment:  <none>
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  copy-system-yaml:
    Container ID:  docker://b7ea3cc0f5d802daeeef8e8417bb8b58903928c6a8d434b98df173e580c5bb11
    Image:         docker.bintray.io/alpine:3.12
    Image ID:      docker-pullable://docker.bintray.io/alpine@sha256:a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/sh
      -c
      echo "Copy system.yaml to /opt/jfrog/pipelines/var/etc"; cp -fv /tmp/etc/system.yaml /opt/jfrog/pipelines/var/etc/system.yaml;
      
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:40:11 -0700
      Finished:     Thu, 08 Oct 2020 10:40:11 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:        100m
    Environment:  <none>
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /tmp/etc/system.yaml from systemyaml (rw,path="system.yaml")
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  wait-for-vault:
    Container ID:  docker://d55a83837b5de938d37549c84048f85ae6c20f5f4d27293728ee5bc4009c9300
    Image:         docker.bintray.io/alpine:3.12
    Image ID:      docker-pullable://docker.bintray.io/alpine@sha256:a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65
    Port:          <none>
    Host Port:     <none>
    Command:
      sh
      -c
      echo "Waiting for Vault to come up..."; until nc -z -w 2 pipelines-pipelines-vault 30100 && echo Vault ok; do
        sleep 2;
      done;
      
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:40:12 -0700
      Finished:     Thu, 08 Oct 2020 10:41:17 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:        100m
    Environment:  <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  pipelines-installer:
    Container ID:  docker://7a58264d4cff55f565aaa2d48e3b77f34599f8a2212beb38ccec83b9df661a14
    Image:         docker.bintray.io/jfrog/pipelines-installer:1.8.0
    Image ID:      docker-pullable://docker.bintray.io/jfrog/pipelines-installer@sha256:70ecd48d061cd1a6c522865df2202d6446f400462ddec55da6488b4501cb9869
    Port:          <none>
    Host Port:     <none>
    Command:
      sh
      -c
      echo "Waiting for RabbitMQ to come up..."; until nc -z -w 2 pipelines-rabbitmq 5672 && echo rabbitmq ok; do
        sleep 2;
      done; echo "Waiting for Redis to come up..."; until nc -z -w 2 pipelines-redis-master 6379 && echo redis ok; do
        sleep 2;
      done; sleep 20; ./pipelines-k8s; echo "Setting router as user for system.yaml"; chown 1117:1117 /opt/jfrog/pipelines/var/etc/system.yaml;
      
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:41:18 -0700
      Finished:     Thu, 08 Oct 2020 10:41:57 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:  100m
    Environment:
      VAULT_TOKEN:                           <set to the key 'token' in secret 'root-vault-secret'>            Optional: false
      PIPELINES_SHARED_DB_CONNECTIONSTRING:  <set to the key 'postgresql-url' in secret 'pipelines-database'>  Optional: false
      PIPELINES_NODE_ID:                     pipelines-pipelines-services-0 (v1:metadata.name)
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  custom-setup:
    Container ID:  docker://c8b5af7a83c611e88e2fa2f8428d365310dc816684e26560f0e7119be6ad723c
    Image:         docker.bintray.io/alpine:3.12
    Image ID:      docker-pullable://docker.bintray.io/alpine@sha256:a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65
    Port:          <none>
    Host Port:     <none>
    Command:
      sh
      -c
      touch /opt/jfrog/pipelines/var/etc/example-custom-setup
    State:          Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:41:58 -0700
      Finished:     Thu, 08 Oct 2020 10:41:58 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:        100m
    Environment:  <none>
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
Containers:
  router:
    Container ID:   docker://87a8b0a364644d1c717ba5496a9eaebae4e65e55b531c61062a6689103860c98
    Image:          docker.bintray.io/jfrog/pipelines-router:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-router@sha256:ea2fb608980765f97ba9f74847d540b3bd480b76ccdd96d4556d9a5a9fa8c0fd
    Port:           8046/TCP
    Host Port:      0/TCP
    State:          Running
      Started:      Thu, 08 Oct 2020 10:41:59 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:  100m
    Environment:
      JF_ROUTER_SERVICEREGISTRY_URL:          http://artifactory-ha-nginx/access
      JF_ROUTER_SERVICEREGISTRY_GRPCADDRESS:  artifactory-ha-nginx
      JF_ROUTER_ENTRYPOINTS_INTERNALPORT:     8046
      JF_ROUTER_ENTRYPOINTS_EXTERNALPORT:     8082
      JF_ROUTER_LOGGING_ROUTER_LOGLEVEL:      DEBUG
      JF_SHARED_NODE_ID:                      pipelines-pipelines-services-0 (v1:metadata.name)
      JF_SHARED_NODE_IP:                       (v1:status.podIP)
      JF_SHARED_SECURITY_JOINKEY:             
      JF_ROUTER_ENCRYPTSYSTEMCONFIG:          true
    Mounts:
      /opt/jfrog/router/var/etc from jfrog-pipelines-folder (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  api:
    Container ID:   docker://ba1611bf851fa53ada8dfd5f7c62a6b1557a870703cb0125fb262631f5559a4e
    Image:          docker.bintray.io/jfrog/pipelines-api:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-api@sha256:1f160965e50f78fd6fbd4a94e6ab000b1705ab2537a28b07023a75bf44d56b0b
    Port:           30000/TCP
    Host Port:      0/TCP
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Completed
      Exit Code:    0
      Started:      Thu, 08 Oct 2020 10:43:32 -0700
      Finished:     Thu, 08 Oct 2020 10:43:34 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:      100m
    Liveness:   http-get http://:api/ delay=20s timeout=10s period=10s #success=1 #failure=10
    Readiness:  http-get http://:api/ delay=20s timeout=10s period=10s #success=1 #failure=10
    Environment:
      PIPELINES_NODE_ID:  pipelines-pipelines-services-0 (v1:metadata.name)
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  www:
    Container ID:   docker://addf999e658360b5a6324f6ecbc8f819361afbc042869817c98b9ad8063899ac
    Image:          docker.bintray.io/jfrog/pipelines-www:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-www@sha256:00e4274d663262fcf167ced83333e9a3ae29e0257e5bee9242f44a940c14b57f
    Port:           30001/TCP
    Host Port:      0/TCP
    State:          Running
      Started:      Thu, 08 Oct 2020 10:41:59 -0700
    Ready:          False
    Restart Count:  0
    Requests:
      cpu:        100m
    Liveness:     http-get http://:www/ delay=20s timeout=10s period=10s #success=1 #failure=10
    Readiness:    http-get http://:www/ delay=20s timeout=10s period=10s #success=1 #failure=10
    Environment:  <none>
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  pipelinesync:
    Container ID:   docker://895f40a4deaed92dd1f9aeb3bada876de7b1a93dfb9b3cff34c0a96660e9aa80
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:33 -0700
      Finished:     Thu, 08 Oct 2020 10:43:34 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  pipelinesync
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  runtrigger:
    Container ID:   docker://d201256655aa75c481c2b638506c0e0387c173f99c363139ce62ee5057e144a7
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:33 -0700
      Finished:     Thu, 08 Oct 2020 10:43:34 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  runtrigger
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  steptrigger:
    Container ID:   docker://9a076933fc2f58679df93217352e165b4a53fb7e661b916b8a20f33b70846758
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:33 -0700
      Finished:     Thu, 08 Oct 2020 10:43:34 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  steptrigger
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  cron:
    Container ID:   docker://d7420e6adc64a17156c438ce97268864c82185e462325c41204c61657bedf86a
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Running
      Started:      Thu, 08 Oct 2020 10:42:01 -0700
    Ready:          True
    Restart Count:  0
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  cron
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  nexec:
    Container ID:   docker://9d39653c297c79043d8cca4683df83d4676a9f04a14e00f19d5373dce6dc20c5
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:34 -0700
      Finished:     Thu, 08 Oct 2020 10:43:35 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  nexec
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  hookhandler:
    Container ID:   docker://408ac01e2977640ac5cd7afdf0b58f11b92a14afc492fb892ba0d58392ab333f
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:34 -0700
      Finished:     Thu, 08 Oct 2020 10:43:35 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  hookhandler
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  marshaller:
    Container ID:   docker://4c4c0d7efc6e704d9c497ebe3fd58974f3748a9ebf70e5d35d699a38a5f516ee
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:34 -0700
      Finished:     Thu, 08 Oct 2020 10:43:36 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  marshaller
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  logup:
    Container ID:   docker://4fd5b565b3ef3752d60c0a0e7a734ccb1f8800b2de7f2a4e9cc4f17bc0a8285e
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:35 -0700
      Finished:     Thu, 08 Oct 2020 10:43:35 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  logup
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
  extensionsync:
    Container ID:   docker://920998aec4a8656330e52066515464bdfbccb91c0968e72c9493748644115cb9
    Image:          docker.bintray.io/jfrog/pipelines-micro:1.8.0
    Image ID:       docker-pullable://docker.bintray.io/jfrog/pipelines-micro@sha256:e517d387c1abb7bc16d0b2a1a558da8605b3adda3d962caa4676371208318cd8
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Thu, 08 Oct 2020 10:43:35 -0700
      Finished:     Thu, 08 Oct 2020 10:43:36 -0700
    Ready:          False
    Restart Count:  4
    Requests:
      cpu:  100m
    Environment:
      COMPONENT:  extensionsync
    Mounts:
      /opt/jfrog/pipelines/var/etc from jfrog-pipelines-folder (rw)
      /opt/jfrog/pipelines/var/log from jfrog-pipelines-logs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from pipelines-token-7cxnz (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  jfrog-pipelines-folder:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:     
    SizeLimit:  <unset>
  jfrog-pipelines-logs:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:     
    SizeLimit:  <unset>
  systemyaml:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  pipelines-system-yaml
    Optional:    false
  pipelines-token-7cxnz:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  pipelines-token-7cxnz
    Optional:    false
QoS Class:       Burstable
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type    Reason     Age    From                                                          Message
  ----    ------     ----   ----                                                          -------
  Normal  Scheduled  3m30s  default-scheduler                                             Successfully assigned default/pipelines-pipelines-services-0 to gke-johnppipelineschartc-default-pool-f2d5c611-lm1b
  Normal  Pulled     3m29s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/alpine:3.12" already present on machine
  Normal  Created    3m29s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container custom-begin-setup
  Normal  Started    3m29s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container custom-begin-setup
  Normal  Pulled     3m28s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/alpine:3.12" already present on machine
  Normal  Created    3m28s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container copy-system-yaml
  Normal  Started    3m28s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container copy-system-yaml
  Normal  Pulled     3m27s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/alpine:3.12" already present on machine
  Normal  Created    3m27s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container wait-for-vault
  Normal  Started    3m27s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container wait-for-vault
  Normal  Pulled     2m21s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/jfrog/pipelines-installer:1.8.0" already present on machine
  Normal  Created    2m21s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container pipelines-installer
  Normal  Started    2m21s  kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container pipelines-installer
  Normal  Started    101s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container custom-setup
  Normal  Created    101s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container custom-setup
  Normal  Pulled     101s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/alpine:3.12" already present on machine
  Normal  Pulled     100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/jfrog/pipelines-router:1.8.0" already present on machine
  Normal  Created    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container router
  Normal  Started    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container router
  Normal  Pulled     100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/jfrog/pipelines-api:1.8.0" already present on machine
  Normal  Created    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container api
  Normal  Started    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container api
  Normal  Pulled     100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/jfrog/pipelines-www:1.8.0" already present on machine
  Normal  Created    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Created container www
  Normal  Started    100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Started container www
  Normal  Pulled     100s   kubelet, gke-johnppipelineschartc-default-pool-f2d5c611-lm1b  Container image "docker.bintray.io/jfrog/pipelines-micro:1.8.0" already present on machine

**Review by Khazinawaz — APPROVED** [review]
LGTM!

File: stable/pipelines/README.md [hunk 2]
```diff
@@ -81,6 +80,21 @@ kubectl create ns pipelines
 helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f pipelines/values-ingress.yaml -f pipelines/values-ingress-passwords.yaml
 ```
 
+### Special Upgrade Notes
+
+While upgrading from Pipelines 1.x to 2.x and above charts, Due to breaking changes in rabbitmq subchart (6.x to 7.x chart version when `rabbitmq.enabled=true`) and postgresql subchart(8.x to 9.x chart version when `postgresql.enabled=true`) please run below manual commands (downtime is required)
```
Comment (rageshkrishna) [inline stable/pipelines/README.md:85]: `Pipelines 1.x to 2.x` might be interpreted as app versions. Can you rephrase this to make it clear that we're talking about chart versions? Maybe something like `While upgrading from Pipelines chart version 1.x to 2.x or above...`

Comment (chukka) [reply stable/pipelines/README.md:85]: Sure

Comment (chukka) [reply stable/pipelines/README.md]: Done

**Review by rahulsadanandan — APPROVED** [review]

File: stable/pipelines/values.yaml [hunk 3]
```diff
@@ -48,9 +89,9 @@ pipelines:
   # version:
 
   ## Artifactory URL - Mandatory
-  jfrogUrl:
+  jfrogUrl: ""
   ## Artifactory UI URL - Mandatory
-  jfrogUrlUI:
+  jfrogUrlUI: ""
```
Comment (eldada) [inline stable/pipelines/values.yaml:94]: I suggest adding an example of what might be `jfrogUrlUI` compared to `jfrogUrl`. I don't know what is the difference.

**Review by eldada — APPROVED** [review]

File: stable/pipelines/values.yaml (hunk 3 above)
Comment (chukka) [reply stable/pipelines/values.yaml:97]: Done

**Review by rimusz — APPROVED** [review]
LGTM
